spec/tbd_tests_spec.rb

```diff
@@ -9806,6 +9806,53 @@
     expect(kfs.size).to eq(3)
   end
 
+  it "can invalidate KIVA inputs (smalloffice)" do
+    TBD.clean!
+    argh = {}
+    translator = OpenStudio::OSVersion::VersionTranslator.new
+    file = File.join(__dir__, "files/osms/in/smalloffice.osm")
+    path = OpenStudio::Path.new(file)
+    model = translator.loadModel(path)
+    expect(model.empty?).to be(false)
+    model = model.get
+
+    # Reset all ground-facing floor surfaces as "foundations".
+    model.getSurfaces.each do |s|
+      next unless s.outsideBoundaryCondition.downcase == "ground"
+      expect(s.construction.empty?).to be(false)
+      construction = s.construction.get
+      expect(s.setOutsideBoundaryCondition("Foundation")).to be(true)
+      expect(s.setConstruction(construction)).to be(true)
+    end
```

Comment:
Above loop is an example of a poor SDK-based KIVA setup. It is imprudent to set "Foundation" as surface boundary conditions without prior validation/adaption of KIVA-targeted layered constructions. In the "smalloffice.osm", the slab-on-grade layered construction holds a massless (or NoMass) material - a _no-no_ for KIVA. Users would need to either delete the massless layer or replace it with a standard (i.e. regular) material.